src/cpp/include/openvino/genai/generation_config.hpp

```diff
@@ -71,6 +73,10 @@ class OPENVINO_GENAI_EXPORTS GenerationConfig {
     size_t max_length = SIZE_MAX;
     bool ignore_eos = false;
     size_t min_new_tokens = 0;
+    
+    // OpenAI exposes stop as an array of strings, but vLLM allows also specifying tokens so we expose both.
+    std::vector<std::string> stop_strings;
```

Comment:
please, add a documentation above the class definition

Reply:
Same for python